docs/source/admin/traffic_ops.rst

```diff
@@ -682,7 +682,7 @@ You must then take the output file :file:`trafficops.csr` and submit a request t
 	sudo cp trafficops.crt /etc/pki/tls/certs
 	sudo chown trafops:trafops /etc/pki/tls/certs/trafficops.crt
 
-If necessary, install the :abbr:`CA (Certificate Authority) certificate's ``.pem`` and ``.crt`` files in ``/etc/pki/tls/certs``.
+If necessary, install the :abbr:`CA (Certificate Authority) certificate's` ``.pem`` and ``.crt`` files in ``/etc/pki/tls/certs``.
```

Comment:
The grave accent (<kbd>\`</kbd>) here should go after the `)`.

The syntax for the `:abbr:` text role is

```rst
:abbr:`ABBREVIATION (An Expansion of the Abbreviation that Tells You What it Means)`
```

Reply:
Fixed.

Reply:
I believe this should be fixed now. Thank you for the explanation.